proximal/examples/test_deconv_sinv_psf.py

```diff
@@ -49,18 +54,22 @@ def adjoint(im_src, im_sim):
     
     return LinOpFactory(im_shape, im_shape, forward, adjoint)
 
-def cost_function(x, im, psf, mu=0.1): 
-    blur_op = my_factory(psf, im.shape)
-    im_blur = blur_op(x)
+def cost_function(x, im, psf, mu=0.1, using_px_conv=True): 
+    if using_px_conv:
+        im_blur = px.conv(psf2_cropped, x)
```

Comment:
Note to self: When the "cropped" PSF is much smaller than the image, shift-and-add algorithm is much more efficient than `fftconvolve2d`. I will get back to this later.